CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v1.15.4 (13/03/2023)
+
+**Common**
+* Changed the log level in case of errors during cache open from `FATAL` to `ERROR`.
```

Comment:
Changed the log level from `FATAL` to `ERROR` in case of errors during opening cache.

Reply:
Changed severity of a log level from FATAL to ERROR in case of errors that appear during opening cache.